mst/about-mst.md

```diff
@@ -114,6 +114,81 @@ Within each configuration plan option, there are several plan types available:
 
 <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/mst-service-plans.png" alt="Managed Service for TimescaleDB selecting a service configuration plan"/>
 
+## High availability
+
+Most minor failures are handled automatically without making any changes to your
+service deployment. This includes failures such as service process crashes, or a
+temporary loss of network access. The service automatically restores normal
+operation when the crashed process restarts automatically or when the network
+access is restored.
+
+However, more severe failure modes such as losing a single node entirely,
+require more drastic recovery measures. Losing an entire node or a virtual
+machine could happen for example due to hardware failure or a severe software
+failure.
+
+A failing node is automatically detected by the MST monitoring infrastructure.
+Either the node starts reporting that its own self-diagnostics is reporting
+problems or the node stops communicating entirely. The monitoring infrastructure
+automatically schedules a new replacement node to be created when this happens.
+
+<highlight type="note">
+In case of database failover, the service URL of your service remains the same,
+only the IP address changes to point at the new master node.
```

Comment:
```suggestion
In case of database failover, the service URL of your service remains the same.
Only the IP address changes to point at the new master node.
```